doc/source/upstream-bug-fixes.rst

```diff
@@ -5,9 +5,27 @@
 =============================================================
```

Comment:
It is not listed here https://www.percona.com/doc/percona-server/5.5/upstream-bug-fixes.html
What is the corresponding LP bug?